lib/measures/tbd/measure.rb

```diff
@@ -186,15 +247,27 @@ def arguments(model)
     write_tbd_json.setDefaultValue(false)
     args << write_tbd_json
 
+    gen_UA_report = OpenStudio::Measure::OSArgument.makeBoolArgument("gen_UA_report", true, false)
+    gen_UA_report.setDisplayName("Generate UA' report")
+    gen_UA_report.setDescription("Compare ∑U•A + ∑PSI•L + ∑KHI•n (model vs UA' reference - see pull-down option below).")
```

Comment:
The sum character might print as □ if this ever runs in a Ruby instance with a strict ASCII character set